packages/integration-sdk-http-client/src/client.ts

```diff
@@ -243,6 +243,17 @@ export abstract class BaseAPIClient {
         });
       } else if (bodyType === 'text' && typeof body === 'string') {
         fmtBody = body;
+      } else if (bodyType === 'urlencoded') {
+        fmtBody = new URLSearchParams();
+        Object.entries(body).forEach(([key, value]) => {
+          if (typeof value !== 'string') {
+            throw new IntegrationError({
```

Comment:
should we provide a softer interface that allows other types but we handle the conversion? Thoughts?

Reply:
there's not too many options left, only binary, and I don't think we will ever upload files ever,  it's ok to keep it like this for now